jr-stree/src/test/java/com/fasterxml/jackson/jr/stree/failing/ReadAsBigDecimal90Test.java

```diff
@@ -13,8 +13,7 @@ public class ReadAsBigDecimal90Test extends JacksonJrTreeTestBase
     public void testReadAsBigDecimal() throws Exception
     {
         JSON json = JSON.builder()
-                .treeCodec(new JacksonJrsTreeCodec())
-                .enable(JSON.Feature.USE_BIG_DECIMAL_FOR_FLOATS)
+                .treeCodec(new JacksonJrsTreeCodec().withFeature(JSON.Feature.USE_BIG_DECIMAL_FOR_FLOATS))
```

Comment:
No, the existing code is the proper way to do it (except for `enable` should probably be added before `treeCodec()` call.

Reply:
done